### ISSUE TITLE
[HOW-TO] close the browser after each NUnit test in the same class (testFixture)

### ISSUE THREAD

**Issue by xuejunh** [Microsoft/EasyRepro#741]
I have many tests in same class (testFicture) but each test need it own different test user so I need to close the browser and login again. How could I close the browser? Or I should have other ways to implement such feature to switch to different user for easyRepro tests in same class? Thanks!

**Comment by AngelRodriguez8008** [comment]
You should handle the Credentials yourself in a Dictionary, List, JSON, XML, Settings File, ect. or Hard-Coded, whereever you want and past this as parameter to the Login methods. By using Private Mode you don't need to take care about closing the browser or log out. That is happening actually in all the provided samples.

Please close the Issues that you got already clarified.

**Comment by xuejunh** [comment]
@AngelRodriguez8008 Do I need to set something to work in the Private mode? Is the Private mode recommended for all the tests?

**Comment by xuejunh** [comment]
@AngelRodriguez8008 Could you point me to the sample code where "That is happening actually in all the provided samples."? Thanks!

**Comment by AngelRodriguez8008** [comment]
Hi @xuejunh, 

Sorry, this PrivateMode was not enable for Edge. I added in this [commit](https://github.com/microsoft/EasyRepro/pull/754/commits/f8720e70ea66b63ece7f294784eee48cb13d7874). This version should be able to get the value from app.config

```xml
    <add key="UsePrivateMode" value="true" />
```

Can you check if this code works for you? You need to compile it by yourself.
Did you have tested in Chrome? Did you know what to expect?

In each tests the Login dialog is shown & the credentials are entered by EasyRepro.
Let me know, if it works.

Thanks & br.

**Comment by AngelRodriguez8008** [comment]
I'm documenting the TestsCreation using the TestsBase class.  Hier is an example, for NUnit just replace the Attributes as shown in comment:
```csharp
    [TestClass] // NUnit =>  [TestFixture]
    public class DemoTestClass_UsingTestBase_ManyTest_SameStart : TestsBase
    {
        [TestInitialize] // NUnit => [SetUp] 
        public override void InitTest() => base.InitTest();

        [TestCleanup] // NUnit => [TearDown]
        public override void FinishTest() => base.FinishTest();

        public override void NavigateToHomePage() => NavigateTo(UCIAppName.Sales, "Sales", "Accounts");

        [TestMethod] // NUnit => [Test]
        public void DemoTest()
        {
            Assert.IsNotNull("Replace this line with your test code");  
            // NUnit has more expressive assertions
            // Assert.That("Replace this line with your test code", Is.Not.Null);
        }
    }
```
NOTE: This code is available for the TestBase in PR #754. Not jet released.
